jcl/src/openj9.jvm/share/classes/com/ibm/jvm/CRIUSupport.java

```diff
@@ -23,17 +22,102 @@
  *******************************************************************************/
 package com.ibm.jvm;
 
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+
 /**
  * CRIU Support API
  */
 public final class CRIUSupport {
 	
-	private static final boolean criuSupportEnabled = isCRIUSupportEnabledImpl();
+	/**
+	 * CRIU operation return type
+	 */
+	public enum CRIUResultType {
```

Comment:
Nested `enum` types are implicitly `static`, but I think it helps to make it explicit.